### PR TITLE
ci(assets): add LV_BUILD_TEST guard

### LINKED ISSUE #5581: fix variable name in v9.0.0
### LVGL version

v9.0.0

### What happened?

There are the same variable names in file `lvgl/examples/assets/emoji/img_emoji_F617.c` and file `lvgl/tests/src/test_assets/test_img_emoji_F617.c`

- file `lvgl/examples/assets/emoji/img_emoji_F617.c`
```c
const LV_ATTRIBUTE_MEM_ALIGN LV_ATTRIBUTE_IMAGE_1F617 uint8_t emoji_F617_map[] = {
...
const lv_image_dsc_t emoji_F617 = {
    .header.cf = LV_COLOR_FORMAT_NATIVE,
    .header.w = 72,
    .header.h = 72,
    .data_size = 5184 * LV_COLOR_DEPTH / 8,
    .data = emoji_F617_map,
};
```
- file `lvgl/tests/src/test_assets/test_img_emoji_F617.c`
```c
const LV_ATTRIBUTE_MEM_ALIGN LV_ATTRIBUTE_IMAGE_1F617 uint8_t emoji_F617_map[] = {
...
const lv_image_dsc_t emoji_F617 = {
    .header.cf = LV_COLOR_FORMAT_NATIVE,
    .header.w = 72,
    .header.h = 72,
    .data_size = 5184 * LV_COLOR_DEPTH / 8,
    .data = emoji_F617_map,
};
```

but In file `lvgl/tests/src/test_assets/test_img_emoji_F617.c` there should be prefix `test_`, such as `test_emoji_F617_map`, `test_emoji_F617`

### How to reproduce?

_No response_

### PR DESCRIPTION
**Pull request by kisvegabor**
### Description of the feature or fix

@XuNeo The images in the `test_images` folder doesn't have the guards. I think it would be uses full to add a `--guard "FOO == 3"` parameter to the image convert. What do you think?

fixes #5581

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Comment by XuNeo** [comment]
The script currently have `LV_BUILD_TEST` and bunch of other macros. 

The issue of global variable name conflict cannot be simply solved by adding more guard. It should be addressed to use unique name like prefixing it with `test_`, `examples_`, `demos_` etc.

File: tests/src/test_assets/test_animimg001.c [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #include "../../../lvgl.h"
 
-#if LV_BUILD_EXAMPLES && LV_USE_ANIMIMG
+#if LV_BUILD_TEST && LV_USE_ANIMIMG
```
Comment (PGNetHun) [inline tests/src/test_assets/test_animimg001.c:3]: Rename in `#endif` line, too

File: tests/src/test_assets/test_kern_one_otf.c [hunk 2]
```diff
@@ -0,0 +1,95 @@
+#include <stdint.h>
```
Comment (PGNetHun) [inline tests/src/test_assets/test_kern_one_otf.c:1]: Missing `#if LV_BUILD_TEST` line

**Review by PGNetHun — CHANGES_REQUESTED** [review]

File: tests/src/test_assets/test_kern_one_otf.c (hunk 2 above)
Comment (kisvegabor) [reply tests/src/test_assets/test_kern_one_otf.c:1]: Fixed

File: tests/src/test_assets/test_animimg001.c (hunk 1 above)
Comment (kisvegabor) [reply tests/src/test_assets/test_animimg001.c:3]: Fixed

**Comment by kisvegabor** [comment]
Thank you @PGNetHun.

@XuNeo 
> The issue of global variable name conflict cannot be simply solved by adding more guard. It should be addressed to use unique name like prefixing it with test_, examples_, demos_ etc.

I think the unique name is not enough:
- In lack of proper linker configuration the unused variables might be linked too 
- It can conflict with the user's image. Not likely, but can happen.

Besides having a `guard` option might be useful for the users too to allow easily switching between images.  We could also use it select images based on `LV_COLOR_DEPTH`.

**Review by PGNetHun — DISMISSED** [review]

**Comment by kisvegabor** [comment]
By passing the merge block after a trivial change.

**Pull request merged**